Sources/KeyboardShortcuts/KeyboardShortcuts.swift

```diff
@@ -45,6 +45,16 @@ public enum KeyboardShortcuts {
 
 		// TODO: Should remove user defaults too.
 	}
+	
+	/**
+	Remove all handlers receiving keyboard shortcuts events.
```

Comment:
Can you also document your use-case for it as mention in the PR description? It's not always clear to users what they would use something for and one or more use-cases can help them realize.

Reply:
Added 👍🏻